devp2p/discovery.py

```diff
@@ -62,7 +66,7 @@ def __init__(self, ip, udp_port, tcp_port=0, from_binary=False):
             # We only want v4 or v6 addresses
             # see https://docs.python.org/2/library/socket.html#socket.getaddrinfo
             ips = [
-                unicode(ai[4][0])
+                str(ai[4][0]) if PY3 else unicode(ai[4][0])
```

Comment:
The use of `unicode` here bugs me.  I'm not sure it's exactly a problem, but `unicode` isn't defined in python 3 so it's weird to have it inlined even though it won't execute in the py3 environment.

This is more of a nitpick but this would be cleaner to just have a `force_text` function and remove the `if/else`.  Again, just a nitpick.  This is totally functional as is.